tox.ini

```diff
@@ -4,10 +4,25 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py26, py27, py33, py34, pypy
+envlist = py26, py27, py33, py34, py36, pypy
 
 [testenv]
-commands = python setup.py test
+setenv =
+    ESEARCH_VERSION=1.7.4
+    ESEARCH_PATH={toxworkdir}
+whitelist_externals =
+    sh
+deps =
+    certifi
+    elasticsearch>=1.3.0,<2.0.0
```

Comment:
We probably don't need to specify our deps here; `setup.py test`, below, will install them according to setup.py's `install_requires`. As a bonus, we don't have to repeat ourselves. :-)

Reply:
Ok, but setup.py install doesn't work with tox.So I created a requirements.txt file that be used by both tox.ini and setup.py